cookbook/testing.md

```diff
@@ -162,30 +159,30 @@ const wrapper = mount(Counter, {
 
 const store = useSomeStore()
 
-// Now this call WILL execute the implementation defined by the store
+// Ahora esta llamada EJECUTARÁ la implementación definida por el almacén
 store.someAction()
 
-// ...but it's still wrapped with a spy, so you can inspect calls
+// ...pero sigue envuelto con un espía, así que puedes inspeccionar las llamadas
 expect(store.someAction).toHaveBeenCalledTimes(1)
 ```
 
-### Specifying the createSpy function
+### Especificación de la función createSpy
 
-When using Jest, or vitest with `globals: true`, `createTestingPinia` automatically stubs actions using the spy function based on the existing test framework (`jest.fn` or `vitest.fn`). If you are using a different framework, you'll need to provide a [createSpy](https://pinia.vuejs.org/api/interfaces/pinia_testing.TestingOptions.html#createspy) option:
+Cuando se utiliza Jest, o vitest con `globals: true`, `createTestingPinia` automáticamente stubs acciones utilizando la función espía basada en el marco de prueba existente (`jest.fn` o `vitest.fn`). Si estás utilizando un framework diferente, tendrás que proporcionar una opción [createSpy](https://pinia.vuejs.org/api/interfaces/pinia_testing.TestingOptions.html#createspy):
```

Comment:
> "Cuando se utiliza o Jest o vitest con `globals: true`, `createTestingPinia` automáticamente stubs acciones utilizando la función espía basada en el marco de prueba existente" 

could be translated as 

> "Cuando se utiliza o Jest o vitest con `globals: true`, `createTestingPinia` automáticamente las acciones stubs usan la funciones espía en base a framework de pruebas existente" 

since is more understandable. Do you agree?

Reply:
sure, I applied change